docker/Dockerfile.full

```diff
@@ -0,0 +1,61 @@
+# The image tagged with the 'full' suffix
+ARG IMAGE_BASE_NAME
+ARG BASE_IMAGE_HASH
+ARG BASE_MITIE_IMAGE_HASH
+
+FROM ${IMAGE_BASE_NAME}:base-mitie-${BASE_MITIE_IMAGE_HASH} as mitie
+
+FROM ${IMAGE_BASE_NAME}:base-${BASE_IMAGE_HASH} as builder
+
+COPY --from=mitie /build/data /build/data
+
+# copy files
+COPY . /build/
+COPY docker/configs/config_pretrained_embeddings_spacy_en_duckling.yml /build/config.yml
+
+# change working directory
+WORKDIR /build
+
+# install dependencies
+RUN python -m venv /opt/venv && \
+  . /opt/venv/bin/activate && \
+  pip install --no-cache-dir -U 'pip<20'
```

Comment:
I'm pretty sure `pip<20` is no longer true and we can dump that. There was an issue in Slack that the latest latest version doesn't work, but `>20` definitely works. Also fine to do in a separate PR though.

Reply:
👍🏻